person.rb

```diff
@@ -0,0 +1,27 @@
+class Person
+  def initialize(age, parent_permission = true, name = 'Unknown')
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :name, :age
+  attr_reader :id
+
```

Comment:
- [ ] Good implementation however, I recommend you to add the `attr_accessor and attr_reader` above `def initialize` because you need to access them inside the program.